packages/components/src/components/button/_button.scss

```diff
@@ -167,12 +176,25 @@
     }
   }
 
-  .#{$prefix}--btn--icon-only {
+  .#{$prefix}--btn.#{$prefix}--btn--icon-only.#{$prefix}--tooltip__trigger {
     @include tooltip--trigger('icon', 'bottom');
+    outline: $button-outline-width solid transparent;
+    outline-offset: -4px;
+  }
+
+  .#{$prefix}--btn.#{$prefix}--btn--icon-only.#{$prefix}--tooltip__trigger:hover
+    svg {
+    fill: red;
```

Comment:
what's this block for? just wondering since it's overwritten by https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/components/button/_mixins.scss#L94

Reply:
whoops that was for testing, nice catch 😅